Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1480,13 +1472,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
         public hide() {
             if (this.repositionTimer) 
                 clearTimeout(this.repositionTimer);
-            this.options.combobox.setAttribute("aria-expanded", "false");
-            this.input.removeAttribute("aria-activedescendant");
+            this.input.setAttribute("aria-expanded", "false");
+            this.input.setAttribute("aria-activedescendant", "");
             this.element && LayoutUtility.Hide(this.element);
             if (this.listItems) {
                 for (const listItem of this.listItems) {
                     listItem.classList.remove(this.CLASSES.ACTIVE);
-                    listItem.setAttribute("aria-selected", "false")
+                    listItem.removeAttribute("aria-selected")
```

Comment:
why do you remove it instead of setting it `false`? same at the other places where it gets removed.